### ISSUE TITLE
Update exhibits to spotlight v3.6.0.beta7

### ISSUE THREAD

**Issue by corylown** [sul-dlss/exhibits#2467]
Blocked by:
- https://github.com/projectblacklight/spotlight/pull/2863 (ID bug was introduced in https://github.com/projectblacklight/spotlight/releases/tag/v3.6.0.beta1 with this PR: https://github.com/projectblacklight/spotlight/pull/2847)
- https://github.com/sul-dlss/exhibits/issues/2470
- https://github.com/sul-dlss/exhibits/issues/2475
- https://github.com/sul-dlss/exhibits/issues/2476
- https://github.com/sul-dlss/exhibits/issues/2477
- https://github.com/sul-dlss/exhibits/issues/2479

**Comment by corylown** [comment]
Superseded by https://github.com/sul-dlss/exhibits/issues/2528
